corc-cascading/src/test/java/com/hotels/corc/cascading/SearchArgumentFactoryTest.java

```diff
@@ -18,11 +18,13 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.sameInstance;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.*;
```

Comment:
Consider fully qualifying all imports so that usages are easy to identify for readers.

Reply:
Yes, please do this.